File: MODUL04/PraktukumC/3.c
#include <stdio.h>

void tambahkurang(int angka1, int angka2) {
    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; i++) {
            printf("%d %d", i, angka2 - i + angka1);
            if (i < angka2) {
                printf(" - ");
            }
        }
    } else {
        for (int i = angka1; i >= angka2; i--) {
            printf("%d %d", i, angka2 - i + angka1);
            if (i > angka2) {
                printf(" - ");
            }
        }
    }
}

int main() {
    int ke1, ke2;
    scanf("%d %d", &ke1, &ke2);
    tambahkurang(ke1, ke2);

    return 0;
}

File: MODUL04/PraktikumPy/3.py
def tambah_kurang(angka1, angka2):
    if angka1 < angka2:
        for i in range(angka1, angka2 + 1):
            print(f"{i} {angka2 - i + angka1}", end="")
            if i < angka2:
                print(" - ", end="")
    else:
        for i in range(angka1, angka2 - 1, -1):
            print(f"{i} {angka2 - i + angka1}", end="")
            if i > angka2:
                print(" - ", end="")

if __name__ == "__main__":
    ke1, ke2 = map(int, input().split())
    tambah_kurang(ke1, ke2)
